src/lib/model/OutboundTransfersModel.js

```diff
@@ -208,30 +208,32 @@ class OutboundTransfersModel {
                     });
 
                     // check we got the right payee and info we need
-                    if(payee.partyIdInfo.partyIdType !== this.data.to.idType) {
+                    if (payee.partyIdInfo.partyIdType !== this.data.to.idType) {
                         const err = new Error(`Expecting resolved payee party IdType to be ${this.data.to.idType} but got ${payee.partyIdInfo.partyIdType}`);
                         return reject(err);
                     }
 
-                    if(payee.partyIdInfo.partyIdentifier !== this.data.to.idValue) {
+                    if (payee.partyIdInfo.partyIdentifier !== this.data.to.idValue) {
```

Comment:
why?

Reply:
Not sure if it is the IDE settings that changed them . I see further down in the code some more "IFs" with a spaces and then up at line 112 "IFs" without spaces. Ill remove the spaces between the If and the condition throughout.